exchange_info_service_test.go

```diff
@@ -50,12 +62,34 @@ func (s *exchangeInfoServiceTestSuite) TestExchangeInfo() {
 
 func (s *exchangeInfoServiceTestSuite) assertExchangeInfoEqual(e, a *ExchangeInfo) {
 	r := s.r()
-	for i := 0; i < len(a.Symbols); i++ {
+	for i, currentSymbol := range a.Symbols {
```

Comment:
since we have mocked the API, the result is certain to be Symbol[1], we could remove this line of `range`, just directly assert symbol instance.
for example:
`r.Len(a.Symbols, len(e.Symbols))
r.Equal(e.Symbols[0].BaseAsset, a.Symbols[0].BaseAsset, "BaseAsset")
...
`

Reply:
But it may be useful for future additions to the test, isn't it?
Someone may add some new tests or more symbols to the response and this code will work.
Otherwise it's ties to the fact currently we mock it with only one symbol

Reply:
ok, just keep it.